Example/Source/StoryboardViewController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  StoryboardViewController.swift
+//  Example
+//
+//  Created by Alex Yanski on 2/7/20.
+//  Copyright © 2020 Touchlane LLC. All rights reserved.
+//
+
+import UIKit
+import CampcotCollectionView
+
+class StoryboardViewController: UIViewController {
+
+    @IBOutlet weak var collectionView: CampcotCollectionView!
+
+    let itemsInRow = 2
+    var itemsInSection: [Int: Int] = [:]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.collectionView.delegate = self
```

Comment:
You can set this up through storyboard too. dataSource is the same.

Reply:
Done